### PR TITLE
odim 3742:Updating the readme  for volume changes

### PR DESCRIPTION
**Pull request by jeevan-kamkar**

### CODE REVIEW

**Review by a-ajith — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Review by skhan-anjum — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by tamanna-k — APPROVED** [review]

File: svc-systems/README.md [hunk 1]
```diff
@@ -977,6 +976,8 @@ curl -i -X DELETE \
 |@Redfish.OperationApplyTimeSupport|Redfish annotation \(optional\)<br> | It enables you to control when the operation is carried out.<br> Supported value is: `OnReset`. Supported values are: `OnReset` and `Immediate`. `OnReset` indicates that the volume will be deleted only after you reset the system.<br> |
 
 
+<br>**NOTE:**<br> Deleted volume will be reflected only after the system resets. To see the change, [reset the computer system](#resetting-a-computer-system).
```
Comment (Bharath-KKB) [inline svc-systems/README.md]: individual addition/deletion will be triggered after reboot. In the meanwhile we cannot add or delete volumes.

File: svc-systems/README.md [hunk 2]
```diff
@@ -934,8 +934,7 @@ curl -i -X POST \
 }
 ```
 
-
-
+<br>**NOTE:**<br> Created volume will be reflected only after the system resets. To see the change, [reset the computer system](#resetting-a-computer-system).
```
Comment (Bharath-KKB) [inline svc-systems/README.md]: individual addition/deletion will be triggered after reboot. In the meanwhile we cannot add or delete volumes.

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
address comments

File: svc-systems/README.md (hunk 1 above)
Comment (jeevan-kamkar) [reply svc-systems/README.md]: updated

File: svc-systems/README.md (hunk 2 above)
Comment (jeevan-kamkar) [reply svc-systems/README.md]: updated

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**